### ISSUE TITLE
[Screen reader - Logic apps - Parameters Tab]: Screen reader is not announcing the information about the error message appearing for the "Method" combo box.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6095]
### Describe the issue

User Experience:  
Visually impaired users who rely on screen reader for navigation would be impacted if the screen reader doesn't announce the error message information which is displayed on screen. The user will not be able to know in which control the error message is appearing and will face challenges in navigating the page.
Note: User credentials should NOT be included in the bug.

Pre-requisites:
- Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed.
- Navigate to Azure control present on the right side and select it.
- Navigate to LogicApp icon present under workspace section, click on tat and select 'Create New Project'.
-  Navigate select stateful workflow, then enter the name of the workflow.
- Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter.
- Navigate to 'Open in current window' and select it.
- A dialog will appear, select 'Yes', now the workflow will open in new window.

Repro Steps: 
- Open Visual Studio app.
- Navigate to any workflow and right click on it, then select 'open designer'.
- Navigate all the controls of the designer screen.
- Now add "HTTP" action and select it from designer screen after adding it.
- New dialog appears, navigate all the controls of 'Parameters' tab.
- Verify if screen reader is not announcing the information about the error message appearing for the "Method" combo box.

Actual Result: 
Screen reader is not announcing the information about the error message appearing for the "Method" combo box i.e when focus moves to the control, screen reader JAWS only announces as "Method edit combo collapsed".

Observation:
Same issue exists with both Narrator and JAWS. Both the screen readers only announce the combo box information but didn't announce the error message information.

Expected result: 
Screen reader should announce the information about the "Method is required" error message appearing for the "Method" combo box.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Chrome, Edge
